src/modules/database.module.ts

```diff
@@ -15,7 +15,7 @@ import { Season } from 'src/entities/season.entity';
                 username: configService.get('DB_USER'),
                 password: configService.get('DB_PW'),
                 database: configService.get('DATABASE'),
-                entities: [Season],
+                entities: [SeasonEntity],
```

Comment:
we can just provide the array of entities here, so we don't need to remember to import each one when we add a new one

Reply:
We can do one better than that; if autoLoadEntities is true, the module will automagically find and load in the entities, we don't even have to list them. The typeOrm.config.ts still has to have them loaded manually, though, I think?